### DISCUSSION TITLE
GFS Forecasting Data Access

### DISCUSSION THREAD

**Discussion by rkalghatgi** [weather-gov/api#616] [General]
Is it possible to use the API to access forecasting data from GFS? We're specifically looking to perform data assimilation tasks using synthetic data to understand how the performance of GFS will change based on our synthetic data. Thanks!

**Comment by akrherz** [comment]
What signature of endpoints are you hoping to see?  One option may be [NCEI's Thredds Server](https://www.ncei.noaa.gov/thredds/catalog/model/gfs.html)  Many folks do HTTP range-bytes requests against the index files on the [FTPPRD server](https://ftp.ncep.noaa.gov/data/nccf/com/gfs/prod/) server.  Others use [nomads](https://nomads.ncep.noaa.gov/).
